cmd/buildctl/main.go

```diff
@@ -28,9 +29,29 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:  "addr",
-			Usage: "listening address",
+			Usage: "buildkitd address",
 			Value: defaultAddress,
 		},
+		cli.StringFlag{
```

Comment:
What would be the use case of this one ? a certificate issued for another address than the specified `addr` ?

Reply:
Imagine you have your certificates generated for a specific addressd and your buildserver is running, but you need to access it directly via IP because your DNS is broken or you have setup some port-forwarding to debug things in for example a k8s deployment.

Then it would be usefull to be able to tell the client which name it should expect in the servers certificate.

Reply:
ok 😛 sounds fair 👼